### ISSUE TITLE
Adoption barrier

### ISSUE THREAD

**Issue by rhysjtevans** [metallb/metallb#544]
Hi Guys,
I'm wanting to implement MetalLB on some of our clusters, there's a blocker around the maturity of the MetalLB project. I totally appreciate that however some additional terminology used on the maturity page which was last updated 2 years ago caused concern.

Is there any chance a more up to date stance can be reviewed and published.

A key phrase that caused concern with the compliance team is 
"If you use it today, you are still an early adopter".

Also if the project maturity rating of 4 could also be elaborated.

**Comment by danderson** [comment]
Good concerns.

Bluntly, the reason I wrote those things had more to do with the bus factor of the project: I was a single maintainer, with limited time or interest in continuing to work with Kubernetes.

We're fixing that right now though, volunteers from 2 companies that specialize in Kubernetes are coming on board as maintainers, and are driving things forward again.

I don't know if we should update the maturity page yet, just out of caution. But hundreds of people and companies are using MetalLB in production, so it can't be that bad ;). I'll defer to the new cohort of maintainers to decide how they would like to advertise the project's maturity.

So, bottom line: MetalLB is pretty mature. The main threat to it was my personal lack of time to work on it, but we're fixing that with new maintainers now.

**Comment by rata** [comment]
I strongly prefer to start small, it doesn't sound reasonable to start by updating the maturity page IMHO. I think we should do it when we gain more confidence and are more familiarized with the project.

But thanks again for being so open, @danderson.

Also, I'm also a user, and I fully support what Dave said regarding it is pretty mature (I'm using it in production in pretty big clusters). But I recognize that page was something that got me scared a little bit when first consider using metallb too.

@rhysjtevans does this clarifies your concern? Feel free to ask if you want to know anything else :)

**Comment by rhysjtevans** [comment]
Hi @danderson @rata - Thanks for your quick response. I thought I'd reply and give you some good news.
Thanks to your quick responses and the git commit history I got approval. Thanks again for the support and for building/supporting a great project! I thought I'd get back in touch when I got the go-ahead. 
Thanks again!

Ref a previous issue - Out of curiosity is there any movement now there's more bandwidth in the project to become a CNCF sandbox/incubator project? I think that would be a great move to socialize the project more which would gain even more support potentially? To be fair I've not looked at the commitments involved in that process it's just a thought.

**Comment by rata** [comment]
> Hi @danderson @rata - Thanks for your quick response. I thought I'd reply and give you some good news.
> Thanks to your quick responses and the git commit history I got approval. Thanks again for the support and for building/supporting a great project! I thought I'd get back in touch when I got the go-ahead.
> Thanks again!

Awesome! Feel free to comment here about your use case: https://github.com/metallb/metallb/issues/5 :-)
 
> Ref a previous issue - Out of curiosity is there any movement now there's more bandwidth in the project to become a CNCF sandbox/incubator project? I think that would be a great move to socialize the project more which would gain even more support potentially? To be fair I've not looked at the commitments involved in that process it's just a thought.

I really don't know about that, like not very familiar with the process nor the benefits/requirements. I think we want to have more people involved in the short term and, AFAIK, there is no other thing planned short term. I'm open to whatever we all we agree, but I'd start small (i.e. not go for that just now) and only consider those things in a later stage. Of couse, @danderson opinion is quite relevant.

Also, I'm a tech person and quite lazy regarding those things. But I have some co-workers that might be more familiarized and can ask in the future.

**Comment by russellb** [comment]
closing because it seems like the discussion here completed
